tests/expr13.py

```diff
@@ -7,3 +7,12 @@ def test_Compare():
     a = 3.3 == 3.3
     a = 3.3 != 3.4
     a = complex(3, 4) == complex(3., 4.)
+
+    # string comparison
+    a = "abc" > "abd"
```

Comment:
```suggestion
    a = "abc" > "abd"
    assert a
```

Reply:
And the same for all the other new tests.

Reply:
What's the use of an `assert` statement here right now? These are not integration tests. It will create an `Assert` node in ASR but, it doesn't actually fail if `a` is False with the above-suggested change.

Reply:
Good point, I forgot this is not actually tested in integration tests.